extensions/azurecore/src/account-provider/providerSettings.ts

```diff
@@ -195,10 +195,10 @@ const usNatAzureSettings: ProviderSettings = {
 				endpoint: 'https://vault.cloudapi.eaglex.ic.gov',
 				azureResourceId: AzureResource.AzureKeyVault
 			},
-			azureLogAnalyticsResource: {
+			azureMonitorLogsResource: {
 				id: SettingIds.ala,
```

Comment:
Do you want to change the setting ID too? (it would be nice to expand that to be `SettingIds.azureMonitorLogs` or something as well too)